### PR TITLE
Review the `EDA` use case page

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Review the EDA use case page.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: components/eda/code/eda-bbe/file.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: 'Files, files as events?'
-description: Ballerina enhances file transfer with FTP listeners, reacting to events on servers. SFTP and FTPS options ensure security. Seamlessly integrate file transfer with Ballerina's secure functionality.
+title: 'Files as events'
```
Comment (anupama-pathirage) [inline components/eda/code/eda-bbe/file.md:2]: The previous title was purposeful. So let's keep it as it is.

File: components/eda/code/eda-bbe/websocket.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: 'WebSocket, TCP socket for the Web'
-description: WebSocket is an excellent choice for message transmission, particularly on the Web. With Ballerina, you not only have the capability to utilize WebSocket as-is but also the flexibility to develop custom subprotocols that align with your organization's specific requirements.
+title: 'WebSocket, TCP socket for the web'
+description: WebSocket is an excellent choice for message transmission particularly on the web. With Ballerina, you not only have the capability to utilize a WebSocket as it is but also the flexibility to develop custom subprotocols that align with your organization's specific requirements.
```
Comment (anupama-pathirage) [inline components/eda/code/eda-bbe/websocket.md:3]:
```suggestion
description: WebSocket is an excellent choice for message transmission particularly on the web. With Ballerina, you you have not only the capability to utilize a WebSocket as it is but also the flexibility to develop custom subprotocols that align with your organization's specific requirements.
```

File: components/eda/code/eda-bbe/file.md (hunk 1 above)
Comment (praneesha) [reply components/eda/code/eda-bbe/file.md:2]: Done.

**Comment by anupama-pathirage** [comment]
@shafreenAnfar Please review.

File: components/eda/code/eda-bbe/file.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Files, files as events?'
-description: Ballerina enhances file transfer with FTP listeners, reacting to events on servers. SFTP and FTPS options ensure security. Seamlessly integrate file transfer with Ballerina's secure functionality.
+description: Ballerina enhances file transfer with FTP listeners reacting to events on servers. SFTP and FTPS options ensure security. Seamlessly integrate file transfer with Ballerina's secure functionality.
```
Comment (anupama-pathirage) [inline components/eda/code/eda-bbe/file.md:3]: I feel the version with a comma is slightly clearer and more readable as it helps to separate the different components of the sentence more distinctly.

File: components/eda/code/eda-bbe/graphql.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 'GraphQL subscription'
-description: Ballerina simplifies GraphQL subscriptions, allowing easy event streaming to frontend applications, enabling effortless event subscriptions and delivery to popular clients.
+description: Ballerina simplifies GraphQL subscriptions, allowing easy event streaming to frontend applications and also enabling effortless event subscriptions and delivery to popular clients.
```
Comment (anupama-pathirage) [inline components/eda/code/eda-bbe/graphql.md:3]: I prefer the one without `and also` : The sentence maintains its clarity and readability without the addition of "and also," making it a smoother read. @shafreenAnfar wdyt?

File: components/eda/code/eda-bbe/file.md (hunk 3 above)
Comment (praneesha) [reply components/eda/code/eda-bbe/file.md:3]: Reverted to the original one.

File: components/eda/code/eda-bbe/graphql.md (hunk 4 above)
Comment (praneesha) [reply components/eda/code/eda-bbe/graphql.md:3]: Reverted to the original one.

Comment (shafreenAnfar) [reply components/eda/code/eda-bbe/graphql.md]: Yeah, I think the same.

**Comment by shafreenAnfar** [comment]
LGTM!

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**